cms/templates/library.html

```diff
@@ -53,7 +55,13 @@ <h3 class="sr">${_("Page Actions")}</h3>
                     <a href="#" class="button new-button new-component-button">
                         <i class="icon fa fa-plus icon-inline"></i> <span class="action-button-text">${_("Add Component")}</span>
                     </a>
-              </li>
+                </li>
+                <li class="action-item action-toggle-preview nav-item">
+                    <a href="#" class="button button-toggle-preview action-button toggle-preview-button">
+                        <i class="icon-refresh fa icon preview-arrow"></i>
```

Comment:
This should be cleaned up to use the following classes "icon fa fa-refresh" (I'm not sure what preview-arrow is used for, and icon-refresh is the older font-awesome integration we have since updated.) 

I would also note that there is currently a height difference on the New Component vs the Show Previews button, though there's no scss in this PR directly so it may be an existing issue. (I see the buttons rendering as expected on the course page.)

Finally, is the "Loading ..." pattern necessary? We don't seem to use this for courses (the button just sort of appears in that case.) I'm fine with adding this loading step, but I would suggest adding it to both libraries and courses. We could also go with not having this button loading pattern in either location.  If we do keep this, it would be nice to try and minimize the 'jump' effect of having this appear a bit after page load, but I have no specific suggestion currently on how to achieve this. Is there anything we could do to minimize this?